config/permissions.tmpl.yml

```diff
@@ -35,6 +35,7 @@
 - ems-type:gce_network
 - ems-type:hawkular
 - ems-type:hawkular_datawarehouse
+- ems-type:monitoring
```

Comment:
Question is if we will want to move existing hawkular stuff under the new manager, like metrics or introduce a new alerting provider